data/helpers.d/utils

```diff
@@ -370,72 +385,75 @@ ynh_get_plain_key() {
 
 # Read the value of a key in a ynh manifest file
 #
-# usage: ynh_read_manifest manifest key
-# | arg: -m, --manifest= - Path of the manifest to read
-# | arg: -k, --key= - Name of the key to find
+# usage: ynh_read_manifest --manifest="manifest.json" --key="key"
+# | arg: -m, --manifest=    - Path of the manifest to read
+# | arg: -k, --key=         - Name of the key to find
+# | ret: the value associate to that key
 #
 # Requires YunoHost version 3.5.0 or higher.
 ynh_read_manifest () {
-	# Declare an array to define the options of this helper.
-        local legacy_args=mk
-        declare -Ar args_array=( [m]=manifest= [k]=manifest_key= )
-        local manifest
-        local manifest_key
-	# Manage arguments with getopts
-	ynh_handle_getopts_args "$@"
-
-	if [ ! -e "$manifest" ]; then
-		# If the manifest isn't found, try the common place for backup and restore script.
-		manifest="../settings/manifest.json"
-	fi
-
-	jq ".$manifest_key" "$manifest" --raw-output
+    # Declare an array to define the options of this helper.
+    local legacy_args=mk
+    local -A args_array=( [m]=manifest= [k]=manifest_key= )
+    local manifest
+    local manifest_key
+    # Manage arguments with getopts
+    ynh_handle_getopts_args "$@"
+
+    if [ ! -e "$manifest" ]; then
+        # If the manifest isn't found, try the common place for backup and restore script.
+        manifest="../settings/manifest.json"
```

Comment:
manifest can be a `.toml` file now!!

Reply:
Right, could be.
But to be fixed on another PR.